docs/source/_templates/sdk_submit_tutorial.rst

```diff
@@ -144,6 +164,18 @@ Optionally, you may pass in header properties in order to override any defaults
 
 {% else %}
 
```

Comment:
The `payload_encoder` is just a client-defined convenience function that takes in arbitrary data and spits out _bytes_. You don't have to use one, but if you don't you must make sure the payload is already encoded as _bytes_ when you pass it to `TransactionEncoder.create`.

In the example in the docs, if you tried to call
```
encoder.create({
        'Verb': 'inc',
        'Name': 'foo',
        'Value': 1})
```
And there were _no payload encoder_ it would throw an error. However the following would work fine:
```
encoder.create(cbor.dumps({
        'Verb': 'inc',
        'Name': 'foo',
        'Value': 1}))
```